charts/hono/values.yaml

```diff
@@ -1001,6 +1003,68 @@ deviceRegistryExample:
     # The password to use for authenticating to the MongoDB instance.
     # password:
 
+  # jdbcBasedDeviceRegistry contains configuration properties specific to the
+  # jdbc based device registry.
+  jdbcBasedDeviceRegistry:
+    # imageName contains the name (excluding registry)
+    # of the container image for the example jdbc based device registry.
+    imageName: eclipse/hono-service-device-registry-jdbc
+    # imageTag contains the tag of the container image to deploy.
+    # If not specified, the value of the honoImagesTag property is used.
+    # imageTag:
+    # containerRegistry contains the name of the container registry to pull
+    # the image from.
+    # If not specified, the value of the "honoContainerRegistry" property is used.
+    # containerRegistry:
+    # registry.jdbc contains the configuration properties for device registry
+    # to connect to the database.
+    registry:
```

Comment:
I think I do not quite understand why we need to configure four connections to the database.
I would have expected to configure one connection over which all read and write operations are being performed, or am I missing something?

Reply:
this is how JDBC based device registry is currently implemented. You can see the same type of configuration in the current integration test: https://github.com/eclipse/hono/blob/13d87d1570970b5b942672e9b7a1370a72a6a678/tests/src/test/resources/deviceregistry-jdbc/application.yml#L26-L51

There is some explanation in https://github.com/eclipse/hono/blob/master/site/documentation/content/admin-guide/jdbc-device-registry-config.md
> Supports different JDBC connections for read-only and read-write operations, to support read-only replicas

Reply:
ah, I see